index.html

```diff
@@ -158,8 +158,15 @@
            observer.disconnect();
       }
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    observer.observe({
+      // subscribe to Resource-Timing, User-Timing, and Server-Timing events
+      entryTypes:
+        ['resource', 'mark', 'measure', 'server'],
+
+      // retrieve the buffered Resource-Timing, User-Timing, and Server-Timing events
+      bufferedTypes:
+        ['resource', 'mark', 'measure', 'server']
```

Comment:
FWIW, I understood from previous discussions about exposing ST *only* through PerfObserver that we intend not to expose it through `getEntries*()`. 

If we are exposing it through `getEntries()` as well, I think we have no choice but to diverge the behavior as @cvazac described, where RT/UT are buffered without time limitations, but ST is not.

FWIW, I think that not exposing it *at all* through `getEntries*()` would be less error-prone, as it will be clear that the only way to get ST data is through PerfObserver. Otherwise, developers might erroneously continue to use `getEntries()` with the same familiar patterns and miss ST entries post onload.